src/Displaz.jl

```diff
@@ -234,28 +251,36 @@ When plotting lines, the `linebreak` keyword argument can be used to break the
 position array into multiple line segments.  Each index in the line break array
 is the initial index of a line segment.
 """
-function plot3d(plotobj::DisplazWindow, position; color=[1,1,1], markersize=[0.1], markershape=[0],
+function plot3d(plotobj::DisplazWindow, position; normal=[0,0,1], color=[1,1,1],
```

Comment:
I think normal should probably be optional: set to `nothing` by default, and only written to the file if the user has overridden it.  This way it doesn't bloat the file size unnecessarily.